systers_portal/templates/common/contact.html

```diff
@@ -0,0 +1,22 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+{% block content %}
+<!-- Facebook, Twitter, Linkedin, Github, Systers keeper: Rose Robinson -->
+  <div class="container col-md-8 col-md-offset-2">
+    <h1>Contact Systers</h1>
+    <hr>
+    <p class="contact-text">
+      Systers is curated by the current <a href="mailto:systers-keeper@systers.org">Systers-keeper</a>, Rose Robinson.
+    </p>
+    <p class="contact-text">
+      You can also connect with Systers on our social networks.
+    </p>
+    <div class="container-fluid contact-social">
+      <a href="https://www.facebook.com/abi.systers"><i class="fa fa-facebook-official fa-2x contact-icon"></i></a>
+      <a href="https://twitter.com/systers_org"><i class="fa fa-twitter fa-2x contact-icon"></i></a>
+      <a href="https://www.linkedin.com/groups?home=&gid=3593"><i class="fa fa-linkedin fa-2x contact-icon"></i></a>
```

Comment:
The `&` symbol generated an HTML validation error. To remove the error, use `&amp;` instead.